.azure-pipelines/pipelines.yml

```diff
@@ -220,6 +253,40 @@ stages:
         rbe: false
         artifactSuffix: ".arm64"
         bazelBuildExtraOptions: "--sandbox_base=/tmp/sandbox_base"
+  - job: package
```

Comment:
ive added the packaging as a separate job after `release` - not sure how optimal that is

im wondering whether to either incorporate it with `release` or conversely to possibly incorporate it with what is currently the `docker` stage and make that to something like `builds`